lute/fs/src/fs.cpp

```diff
@@ -334,6 +382,50 @@ int fs_rmdir(lua_State* L)
     return 0;
 }
 
+int fs_stat(lua_State* L)
+{
+    const char* path = luaL_checkstring(L, 1);
+
+    uv_fs_t stat_req;
+    int err = uv_fs_stat(uv_default_loop(), &stat_req, path, nullptr);
+
+    if (err)
+        luaL_errorL(L, "%s", uv_strerror(err));
+
+    lua_newtable(L);
```

Comment:
We can allocate for this too, it's 6 record fields.

Reply:
done